cedar-policy/CHANGELOG.md

```diff
@@ -54,6 +54,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   valid scope constraint. (#818, resolving #563)
 - Fixed policy formatter reordering some comments around if-then-else and
   entity identifier expressions. (#861, resolving #787)
+- The formatter will now fail with an error if it changes a policy's semantics. (#865)
```

Comment:
should we instead list this in 3.1.4?  I think it's generally assumed that 3.2.0 would include all the changes from 3.1.4, especially since it will release chronologically after 3.1.4?

Reply:
Hm, fair point. I hadn't added 2.4.6 or 3.1.4 to this changelog. I'll do that now.

Reply:
Added. Does it look weird to have the same changelog entry under 2.4.6 and 3.1.3?

Reply:
Seems fine IMO

Reply:
I also think it's fine to be duplicated